src/main/java/net/openhft/chronicle/network/internal/SocketExceptionUtil.java

```diff
@@ -20,9 +20,20 @@ private SocketExceptionUtil() {
      * @return true if the exception is one that signifies the connection was reset
      */
     public static boolean isAConnectionResetException(IOException e) {
-        String message = e.getMessage();
-        return message != null &&
-                (message.equals("Connection reset by peer")
-                        || e instanceof SocketException && message.equals("Connection reset"));
+        return isALinuxJava12OrLessConnectionResetException(e)
```

Comment:
my word!

Reply:
Write Once Run Anywhere? ;)

Reply:
Annoying. Connection reset seems like a strong enough concept to warrant its own IOException subclass.